squad/core/migrations/0080_user_subscription.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.14 on 2018-08-02 10:37
+from __future__ import unicode_literals
+
+from django.conf import settings
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('core', '0079_init_cache_test_run_counts'),
```

Comment:
somehow we ended up with 2 identical migrations with only difference in dependency. This needs to be resolved (only one migration is needed) and rebased on current master.